baseapp/testproject/urls.py

```diff
@@ -7,4 +8,5 @@
 
 urlpatterns = [
     path("graphql", GraphQLView.as_view(graphiql=True)),
+    path("admin/", admin.site.urls),
```

Comment:
Added this so I can access the admin when running the testproject independently.